playground/TestPlatform.Playground/TestPlatform.Playground.csproj

```diff
@@ -2,6 +2,9 @@
   <PropertyGroup>
     <TestPlatformRoot Condition="$(TestPlatformRoot) == ''">..\..\</TestPlatformRoot>
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
+    <!-- MSB3270 Suppress warnings about testhost being x64 (AMD64)/x86 when imported into AnyCPU (MSIL) projects. -->
+    <!-- MSB3276 Suppress warnings about conflicts between different versions of the same dependent assembly -->
+    <MSBuildWarningsAsMessages>$(MSBuildWarningsAsMessages);MSB3270;MSB3276</MSBuildWarningsAsMessages>
```

Comment:
I am wondering if it could be better to have these declared in the "global" target with a condition and only add the condition property to the projects (something like `<SuppressTestHostAssemblyWarnings>true</SuppressTestHostAssemblyWarnings>`).

Reply:
Ideally each of those suppressions would be placed directly on the project reference, but that does not work. Probably for the same reason as the NU1702 warning not being suppressed. I did not want to over complicate this, because then it is not very consistent without the NU1702, and understanding what is happening is easy if you look directly at the project. For SuppressTestHostAssemblyWarnings I would have to look at the definition to know what that means. Imho it is good to go as is. But if you really prefer the other solution I can change it.

Reply:
I am fine with current implementation, just wanted to suggest a different path in case that wasn't considered.

Reply:
It wasn't and thank for the comment, it helped me to think about my reasoning further :)